src/factory-in-proc.js

```diff
@@ -131,6 +135,19 @@ class FactoryInProc {
     const node = new Node(options)
 
     series([
+      (cb) => node.exec.once('ready', cb),
+      (cb) => {
+        try {
+          node.repo._isInitialized(() => {
+            node.initialized = true
+            cb()
```

Comment:
Do not call callbacks within try blocks, it results on any error in the future that gets thrown (in any part of the code) to get caught by this try block

Reply:
Great! That's good to know